node/src/bootstrapper.rs

```diff
@@ -386,8 +399,13 @@ impl BootstrapperConfig {
             port_configurations: HashMap::new(),
             data_directory: PathBuf::new(),
             node_descriptor: NodeDescriptor::default(),
-            main_cryptde_null_opt: None,
-            alias_cryptde_null_opt: None,
```

Comment:
These two fields were used by the `--fake-public-key` parameter; but since we now have `cryptde_pair` in here, we don't need them anymore.